packages/react-aria-components/src/HiddenDateInput.tsx

```diff
@@ -50,7 +50,14 @@ export function useHiddenDateInput(props: HiddenDateInputProps, state: DateField
     isDisabled,
     name
   } = props;
-  let {visuallyHiddenProps} = useVisuallyHidden();
+  let {visuallyHiddenProps} = useVisuallyHidden({
+    style: {
+      // Prevent page scrolling.
+      position: 'fixed',
```

Comment:
Should it be `relative`? It's no longer positioned near the visible element: https://reactspectrum.blob.core.windows.net/reactspectrum/a85e685d1afeeb4c3310e84e24b9fc3b56508ef6/docs/react-aria/Select.html#example